apps/storefront/src/pages/Dashboard/index.test.tsx

```diff
@@ -389,6 +390,15 @@ describe('when the user is associated with a company', () => {
         },
       }),
     );
+    const deleteCartReturn = vi.fn().mockReturnValue({
+      data: {
+        cart: {
+          deleteCart: {
+            deletedCartEntityId: '1',
+          },
+        },
+      },
```

Comment:
if we don't use the response, which we don't seem to be using, we shouldn't include it in the mock.

We can ensure the request was correct by using `when`:

```tsx
when(deleteCartReturn)
  .calledWith(stringContainingAll('deletedCartEntityId: 1')) 
  .thenReturn({});
```

makes sure that setting the wrong `cartId` makes the test fail.